### DISCUSSION TITLE
Open a linked note in the same tab

### DISCUSSION THREAD

**Discussion by fedorovpishet** [notaapp/talk#218] [Feature Requests]
What about adding a preferences option — open a linked note in the same tab? 

What's the deal. I use Nota to make notes by Zettelkasten method. So I got very much linked notes. And when I serf through my notes and open them one by one I get SO many opened tabs! It's very uncomfortable to close them all 😢 

I guess it can be useful to add an option to open a linked note in the same tab and in the new tab by hmmm... shift+command+click. What do you think about it?

Look! I start from my start note to open note about Homebrew installation. And I got ELEVEN opened tabs 😄 

<img width="1246" alt="image" src="https://user-images.githubusercontent.com/90445152/166723511-8e563e77-5c85-4ca2-af1c-cb3ab04f0db4.png">

**Comment by astoilkov** [comment]
We have the same problem. We're already exploring ways to improve on this. One of which is opening a link in the same tab. If you have more ideas on how we can solve the problem with too many opened tabs, let us know.

**Reply by fedorovpishet** [reply]
How's it going?)

**Reply by astoilkov** [reply]
We've already created two private experiments that try to solve the problem:
- **Browser-like tabs.** Tabs that work like in the browser. You click on a link and it opens in the same tab. You ⌘+click and it opens in a new tab.
- **Right sidebar with contextual information and editor.** It shows the link you are over. If you aren't over a link, we show contextual information useful for the current document.

We may introduce the right sidebar as an experiment in the next release.

We've been exploring other alternatives but we don't like them as much.

**Comment by work-leonid** [comment]
- by default open the link in the same tab
- if `click` + `shift/cmd/alt` open the link in the tab

Or add settings like in safari
<img width="474" alt="image" src="https://user-images.githubusercontent.com/8654430/166965042-fe037dc0-6ce2-43a6-8793-222f0f096172.png">

**Comment by vladstudio** [comment]
+1 for Browser-like tabs! As a new user, I was surprised to click a link and see it opened in a new tab. I expected opening in the same tab as a default behavior.

**Reply by astoilkov** [reply]
The next version will probably include improved tab management. Stay tuned. I hope all people in this conversation can join with feedback when we release them.
